src/server/worldserver/worldserver.conf.dist

```diff
@@ -4037,6 +4038,23 @@ PacketSpoof.BanDuration = 86400
 #
 ###################################################################################################
 
+###################################################################################################
+# MISC ANTI-CHEAT SETTINGS
+#
+# Settings to tweak various anti cheat measures.
+#
+#    AntiCheats.PendingMoveChangesTimeoutTime
+#        Description: Time each client has to acknowledge a movement change until they are kicked.
+#        Note:        A lower time will give cheaters less advantage when using a very specific
+#                     type of hack. However, the cost is that innocent but laggy players might
+#                     sometimes get kicked.
+#        Default:     2000 (2 seconds)
+
+AntiCheat.PendingMoveChangesTimeoutTime = 5000
```

Comment:
can we have this disabled by default ?

Reply:
Maybe we could change this right before the next release. But before that, for the sake of testing, I would suggest to keep this enabled by default. A player getting kicked is an instant feedback that some assumption in the code is wrong or that there is some kind of bug. It would be nice to catch these issues early.

Reply:
I'd like to get the issues caused by recent prs fixed first, then we can experiment again with users and ask feedback about new features.

Reply:
If i'm not mistaken, all the issues created by https://github.com/TrinityCore/TrinityCore/pull/26348 or https://github.com/TrinityCore/TrinityCore/pull/26538 are either fixed already or they are fixed by this PR. 

However, this PR doesn't just fix issues created recently, it is the first step into making player movement smoother. That's why user feedback on this would be cool.

Reply:
Would it be ok to leave it disabled by default for some time and then we change it ? we can find other ways to get feedback, like actively asking users to enable the setting, but I would prefer this feature to be opt-in for a while (which would allow to merge this PR earlier as we don't have to test every case).

Maybe like merging the PR, letting 1 month pass with no issues related to it reported, then we can enable it by default ? I can enable it on the test realm too and users can test it there if they want to help.

Reply:
Sure let's do that. I'll make the kick disabled by default, right before merging the PR. I'll keep it enable while the PR is open though, to get as much as feedback as possible, if you don't mind

Reply:
is that something that can be implemented with "when AntiCheat.PendingMoveChangesTimeoutTime is set to 0 don't kick, > 0 kick" ?

Reply:
Yes, that's what I was thinking too